extensions/wikia/DesignSystem/DesignSystemHelper.class.php

```diff
@@ -87,7 +87,7 @@ private static function getCachedSvg( $name ) {
 	 * @return string
 	 */
 	public static function renderText( $fields, $recursionDepth = 0 ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 6, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)